Microsoft.Identity.Web/AuthorizeForScopesAttribute.cs

```diff
@@ -72,10 +73,15 @@ public override void OnException(ExceptionContext context)
                             throw new InvalidOperationException($"The {nameof(ScopeKeySection)} is provided but the IConfiguration instance is not present in the services collection");
                         }
 
-                        Scopes = new string[] { configuration.GetValue<string>(ScopeKeySection) };
+                        incrementalConsentScopes = new string[] { configuration.GetValue<string>(ScopeKeySection) };
                     }
 
-                    var properties = BuildAuthenticationPropertiesForIncrementalConsent(Scopes, msalUiRequiredException, context.HttpContext);
+                    if (Scopes != null && Scopes.Length > 0 && incrementalConsentScopes != null && incrementalConsentScopes.Length > 0)
+                    {
+                        throw new InvalidOperationException("no scopes provided here...");
+                    }
+
+                    var properties = BuildAuthenticationPropertiesForIncrementalConsent(incrementalConsentScopes, msalUiRequiredException, context.HttpContext);
```

Comment:
@jennyf19 :I don't see where `incrementalConsentScopes` gets assigned the value of `Scopes` (when the developer uses the scopes directly in the attribute, rather than from the  config file.